instrumentation/grpc-1.5/testing/grpc-1.5-testing.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
+  id "java-library"
+  id "com.google.protobuf" version "0.8.13"
+}
+
+apply from: "$rootDir/gradle/java.gradle"
+
+def grpcVersion = '1.5.0'
+
+protobuf {
+  protoc {
+    // Download compiler rather than using locally installed version:
+    artifact = 'com.google.protobuf:protoc:3.3.0'
+  }
+  plugins {
+    grpc { artifact = "io.grpc:protoc-gen-grpc-java:${grpcVersion}" }
+  }
+  generateProtoTasks {
+    all()*.plugins { grpc {} }
+  }
+}
+
+dependencies {
+  compileOnly(project(':instrumentation:grpc-1.5:library'))
```

Comment:
it's amazing to me that this does not create a cyclic dependency

the test that uses `GrpcHelper` does look a bit suspicious (using the same method to verify the status as was used to map the status), maybe better to add an extra spock data column to those test for otelStatus?

Reply:
Previously I was avoiding having to duplicate the statuscode mapping in the logic, its unit test, and here too. Now it's always one status though so no reason to use the helper :)